api/RTCIceCandidate.json

```
@@ -124,6 +124,253 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "candidateInfo_parameter": {
+          "__compat": {
+            "description": "<code>candidateInfo</code> parameter",
+            "spec_url": "https://w3c.github.io/webrtc-pc/#dom-rtcicecandidateinit",
+            "support": {
+              "chrome": {
+                "version_added": true
+              },
+              "chrome_android": {
+                "version_added": true
+              },
+              "edge": {
+                "version_added": "≤18"
+              },
+              "firefox": {
+                "version_added": "22"
+              },
+              "firefox_android": {
+                "version_added": true
+              },
+              "ie": {
+                "version_added": false
+              },
+              "opera": {
+                "version_added": true
+              },
+              "opera_android": {
+                "version_added": true
+              },
+              "safari": {
+                "version_added": "11"
+              },
+              "safari_ios": {
+                "version_added": "11"
+              },
+              "samsunginternet_android": {
+                "version_added": true
+              },
+              "webview_android": {
+                "version_added": true
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          },
+          "candidateInfo_candidate_parameter": {
+            "__compat": {
+              "description": "<code>candidateInfo.candidate</code> parameter",
+              "spec_url": "https://w3c.github.io/webrtc-pc/#dom-rtcicecandidateinit-candidate",
+              "support": {
+                "chrome": {
+                  "version_added": true
+                },
+                "chrome_android": {
+                  "version_added": true
+                },
+                "edge": {
+                  "version_added": "≤18"
+                },
+                "firefox": {
+                  "version_added": "22",
+                  "notes": "Prior to Firefox 68, the <code>candidate</code> property was required; Firefox 68 and later make it optional per a mid-2017 specification update."
```

This note's a bit complicated for what it's saying. How about this (with the caveat that the name of the parameter might be different)?

```suggestion
                  "notes": "Before Firefox 68, <code>candidateInfo.candidate</code> is required."
```

---

Sure!